registration.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+\Magento\Framework\Component\ComponentRegistrar::register(
+    \Magento\Framework\Component\ComponentRegistrar::MODULE,
+    'VinaiKopp_EavOptionSetup',
+    __DIR__
+);
```

Comment:
Not a blocker, but I prefer the `ComponentRegistrar` to be imported instead of always using the fully qualified class name. I'll change that after the merge if you don't feel like doing the change.